src/lib/cli/browserslist-generator.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * @fileoverview Generates a valid browserslist config.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+import * as inquirer from 'inquirer';
+import { debug as d } from '../utils/debug';
+
+const debug = d(__filename);
+
+export const generateBrowserslistConfig = async () => {
+    debug('Initiating browserslist config generator');
+
+    const browserslistConfig = [];
+    const addBrowsersListOptions = [
+        { name: 'Use Default browsers', value: 'default' },
+        { name: 'Add the last 2 versions of each browser', value: 'last2' },
+        { name: 'Add by percentage of usage statistics. (e.g., `>= 5%`)', value: 'usage' },
+        { name: 'Add by specific browser type and version number range. (e.g., `IE 6-8`)', value: 'type' }
+    ];
+    const browsers = ['Chrome', 'Edge', 'Explorer', 'Firefox', 'Opera', 'Safari'];
```

Comment:
Providing only these browser kinda hints that we only care about desktop browsers. Mobile is a big thing nowadays, and [mobile browsers such as Safari for iOS, UC, Samsung Internet, etc.](http://gs.statcounter.com/), have quite a significant market share. 

Also, in general I personally don't think this is the way to go, we are over complicating things here.

---

I think we should have 2 options: 

* Use default query.
* Specify custom query (+ a link to the [browserslist docs](https://github.com/ai/browserslist#queries))                              

@sonarwhal/contributors :arrow_up: